lib/sensors/src/led_driver.rs

```diff
@@ -0,0 +1,247 @@
+/// LED Driver
+
+/// When setting colours and brightness of LEDs, use the set_led_colours() function.
+///     LEDs needing to be set are controlled by the addresses in either LED_CONFIG0 or LED_CONFIG1 registers
+///     These addresses can be toggled with hexadecimal (E.G: registers 11101000 are set by the hex value E8)
+
+/// link to data sheet (https://www.ti.com/lit/ds/symlink/lp5036.pdf?HQS=dis-mous-null-mousermode-dsf-pf-null-wwe&ts=1698441544495&ref_url=https%253A%252F%252Fwww.mouser.co.uk%252F)
+///
```

Comment:
Good documentation, but move it to above `pub struct LedDriver...` below

Reply:
Done + added additional documentation.